include/psa/crypto_se_driver.h

```diff
@@ -114,10 +114,14 @@ typedef psa_status_t (*psa_drv_se_init_t)(psa_drv_se_context_t *drv_context,
                                           void *persistent_data,
                                           psa_key_lifetime_t lifetime);
 
+#ifdef __DOXYGEN_ONLY__
+/* Mbed Crypto defines this type in crypto_extra.h because it is also
+ * available to applications through implementation-specific extensions. */
```

Comment:
Is it worth a comment to indicate that this type is also defined in `crypto_types.h` - and the definitions need to be identical?

Reply:
I'd prefer to create a new header file. I made this hack to avoid having to mess with build scripts.

Reply:
For the time being, I'm not adding a header file. I think I'll just wait until the header files are no longer used to generate the documentation so this becomes moot.